charts/frr-k8s/values.yaml

```diff
@@ -26,6 +26,9 @@ prometheus:
   # port frr-k8s will listen on for metrics
   metricsPort: 7572
 
+  # port the frr-k8s validating webhook will listen on for metrics
+  whmetricsPort: 7672
+
   # if set, enables rbac proxy on frr-k8s to expose
   # the metrics via tls.
   secureMetricsPort: 9140
```

Comment:
the setup with rback proxy, is it tested? the rbac proxy will also be hostnetwork

Reply:
you can check how the CI is invoked -> deploy-helm which sets a bunch of helm parameters. Looking at the chart, you can see that it's installed.

Reply:
or even better, the webhook doesn't come with the kube-rbac proxy

Reply:
the webhook is not selected by the service of the service monitor, it's safe to just disable the port then.